content/js/access-control.md

```diff
@@ -1 +1,119 @@
-Coming soon
+<a name='acl-type'></a>
+## Access Type
+
+There are two types of access for each record: read and write.
+- Read access allows querying and fetching records, which includes getting
+  all the fields of records as well as the access control settings.
+- Write access allows saving and deleting records, which includes adding,
+  updating and removing all fields of records as well as access control
+  settings for the record.
+
+However, even with write access, ownership of records cannot be changed.
+
+<a name='acl-default'></a>
+## Default ACL Settings
+
+By default, read access is granted to the public and write access is granted
+only to owner of the record. You can change this behavior as well:
+
+``` javascript
+var acl = new skygear.ACL();
+// choose one setting among the three
+acl.setPublicNoAccess();
+acl.setPublicReadOnly(); // default
+acl.setPublicReadWriteAccess();
+skygear.setDefaultACL(acl); // will change skygear.defaultACL
+
+skygear.defaultACL.hasPublicReadAccess(); // default true
+skygear.defaultACL.hasPublicWriteAccess(); // default false
+```
+
+
+<a name='acl-user'></a>
+## ACL by User
+
+Suppose you have three user objects: `enemy`, `visitor` and `friend`.
```

Comment:
These names do not sound like proper `users` but relationship or user roles. You may want to have names like `Bob`, `Mary`, etc.